### DISCUSSION TITLE
DUO-Connect

### DISCUSSION THREAD

**Discussion by Csdcampbell** [Homebrew/discussions#3157] [Casks]
Hello, I've successfully installed DUO-Connect via the cask, but I'm not sure where it installed and it does not show in applications. However, I'm more curious if there is a way to include the 3 security keys in the script somehow so it's ready to point to our specific DUO instances.

**Comment by gromgit** [comment]
> Hello, I've successfully installed DUO-Connect via the cask, but I'm not sure where it installed and it does not show in applications. 

Casks generally just download and run the original software installer, while internally recording the fact that the software was installed. If you need to know where cask-installed software got installed, you really should be reading the software's documentation, or asking on the upstream forums.

> However, I'm more curious if there is a way to include the 3 security keys in the script somehow so it's ready to point to our specific DUO instances.

You really don't want to mess with cask files unless you know what you're doing, because any changes you make may conflict with future updates.

**Reply by Csdcampbell** [reply]
Thanks for the reply. It's not often I have to install DUO-Connect so I don't recall off the top of my head where it installs. I'll try to install it manually and track that aspect down

The issue with DUO is it has 3 keys to incorporate into it so it creates a link to the tenant and checks the tenants user accounts against it. I'm hoping there is a way to add those. It seems I may be able to automate replacing the plist file DUO checks against with a configured copy.
